### PR TITLE
fix: get dextrose web working

### PR DESCRIPTION
**Pull request by rjanjua**
Dextrose hasn't really worked on the web. This PR aims to fix that. 

I have updated the fructose version to fix an issue relating do dependecy of dependencies weirdness.

I have updated react-apollo to the latest release candidate to fix the 'getDataFromTree' issue we see in dextrose - which was a known bug in react-apollo a few versions ago.

I have also removed the dextrose clean and generate stories commands from the following packages as they use the storybook addon knobs - dextrose doesn't work with knobs:
  
- video-label
- slice

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -110,7 +110,8 @@
     "@storybook/react": "3.3.14",
     "@storybook/react-native": "3.3.14",
     "@times-components/depend": "*",
-    "@times-components/fructose": "3.1.14",
+    "@times-components/fructose": "3.2.2",
+    "babel-loader": "*",
```
Comment (nikhedonia) [inline package.json]: can we lock it down to `7.1.4` ?

File: packages/depend/package.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
       "!**/packages/depend/get-packages.js"
     ],
     "transformIgnorePatterns": [
-      "node_modules/(?!(react-native|react-apollo-temp|@times-components)/)"
+      "node_modules/(?!(react-native|react-apollo|@times-components)/)"
```
Comment (nikhedonia) [inline packages/depend/package.json]: would it still work if we remove react-apollo from the list?
We used to depend on a untranspiled file but I recall that I removed that dependency in one of my PRs

File: packages/provider/connect.js [hunk 3]
```diff
@@ -30,14 +30,13 @@ export const makeGraphqlOptions = (
...
-    data: { error, loading, refetch, retry, ...result },
+    data: { error, loading, refetch, ...result },
     children,
     ...props
   }) =>
     children({
       error,
       refetch: () => {
-        retry(); // FIXME: remove this after react-apollo fixes https://github.com/apollographql/apollo-client/issues/2513
         refetch();
```
Comment (nikhedonia) [inline packages/provider/connect.js]: this is redundant. Use a shorthand like in case of error

File: package.json (hunk 1 above)
Comment (rjanjua) [reply package.json]: oh yeah, my bad

File: packages/provider/connect.js (hunk 3 above)
Comment (rjanjua) [reply packages/provider/connect.js]: i'm not sure what you mean

Comment (rjanjua) [reply packages/provider/connect.js]: oh, yeah, don't worry - I understand now

File: packages/depend/package.json (hunk 2 above)
Comment (rjanjua) [reply packages/depend/package.json]: no idea, but will try

File: packages/utils/graphql.js [hunk 4]
```diff
@@ -2,8 +2,8 @@
 
 import React, { Component } from "react";
 import ApolloClient from "apollo-client";
-import { ApolloProvider } from "react-apollo-temp";
-import { MockLink } from "react-apollo-temp/lib/test-utils";
```
Comment (nikhedonia) [inline packages/utils/graphql.js]: Good spot! I wonder how that sneaked back into our codebase

Comment (craigbilner) [reply packages/utils/graphql.js]: it was never removed cos of the usual umd issues

File: packages/provider/connect.js [hunk 5]
```diff
@@ -36,7 +36,7 @@ const connectGraphql = (query, propsToVariables) => {
   }) =>
     children({
       error,
-      refetch,
+      refetch: () => refetch(), // using shorthand causes a rect-native error
```
Comment (rjanjua) [inline packages/provider/connect.js]: so I tested this in storybook, and the shorthand version results in a strange error: 

screen shot 2018-03-09 at 17 10 12

File: packages/utils/graphql.js (hunk 4 above)
Comment (nikhedonia) [reply packages/utils/graphql.js]: You are right, was thinking about my older PR: 
https://github.com/newsuk/times-components/pull/613/files

**Pull request merged**